packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -3,15 +3,63 @@
 // found in the LICENSE file.
 
 #import "FLTVideoPlayerPlugin.h"
+
 #import <AVFoundation/AVFoundation.h>
 #import <GLKit/GLKit.h>
-#import "AVAssetTrack+Utils.h"
+
 #import "messages.g.h"
 
 #if !__has_feature(objc_arc)
 #error Code Requires ARC.
 #endif
 
+/**
+ * Note: https://stackoverflow.com/questions/64161544
+ * `AVAssetTrack.preferredTransform` can have wrong `tx` and `ty`
+ * on iOS 14 and above. This method provide a standardized transform
+ * according to the orientation of the track.
+ */
+static CGAffineTransform FLTGetStandardizedTransformForTrack(AVAssetTrack *track) {
```

Comment:
A static function is internal to the compilation unit (this file), so can't be access from the test. You need a public declaration of this in a header, just as you do for ObjC classes you want to use in other files.

You should have a separate AVAssetTrackUtils.h and AVAssetTrackUtils.m where you declare and implement this, just as you did when you had it as a category, just using a (non-static) function like this instead of a category.

Reply:
Thanks! That helps a lot